### ISSUE TITLE
Flag "editable" has no effect

### ISSUE THREAD

**Issue by MaxTru** [bpmn-io/bpmn-properties-panel#177]
__Describe the Bug__

"editable" has no effect

__Steps to Reproduce__

1. Have a template like this:
```
{
    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.4.0/resources/schema.json",
    "name": "Mail Task",
    "id": "com.camunda.example.MailTask",
    "symbold": "someDataUri",
    "version": 2,
    "appliesTo": [
      "bpmn:ServiceTask"
    ],
    "properties": [
      {
        "label": "Implementation Type",
        "type": "String",
        "value": "com.mycompany.MailTaskImpl",
        "editable": false,
        "binding": {
          "type": "property",
          "name": "camunda:class"
        }
      },
      {
        "label": "Sender",
        "type": "String",
        "binding": {
          "type": "camunda:inputParameter",
          "name": "sender"
        },
        "constraints": {
          "notEmpty": true,
          "pattern": {
            "value": "^[A-z0-9._%+-]+@[A-z0-9.-]+\\.[A-z]{2,}$",
            "message": "Must be a valid email."
          }
        }
      },
      {
        "label": "Receivers",
        "type": "String",
        "binding": {
          "type": "camunda:inputParameter",
          "name": "receivers"
        },
        "constraints": {
          "notEmpty": true
        }
      },
      {
        "label": "Template",
        "description": "By the way, you can use <a href=\"https://freemarker.apache.org/\">freemarker templates</a> here",
        "value": "Hello ${firstName}!",
        "type": "Text",
        "binding": {
          "type": "camunda:inputParameter",
          "name": "messageBody",
          "scriptFormat": "freemarker"
        },
        "constraints": {
          "notEmpty": true
        }
      },
      {
        "label": "Result Status",
        "description": "The process variable to which to assign the send result to",
        "value": "mailSendResult",
        "type": "String",
        "binding": {
          "type": "camunda:outputParameter",
          "source": "${ resultStatus }"
        }
      },
      {
        "label": "Send Async?",
        "type": "Boolean",
        "value": true,
        "binding": {
          "type": "property",
          "name": "camunda:asyncBefore"
        }
      }
    ]
  }
```
2. Apply the template
3. See that respective field is still editable

__Expected Behavior__

"editable" flag has effect on respective field

__Environment__

Tested based on https://github.com/bpmn-io/bpmn-properties-panel/pull/161

**Comment by barmac** [comment]
I was able to reproduce this.

**Comment by barmac** [comment]
Just found that only the TextField implements `disabled` property. This means we will have to release a new version of `@bpmn-io/properties-panel` to fix this.

**Comment by MaxTru** [comment]
> Just found that only the TextField implements `disabled` property. This means we will have to release a new version of `@bpmn-io/properties-panel` to fix this.

#lastRelease

**Comment by barmac** [comment]
> > Just found that only the TextField implements `disabled` property. This means we will have to release a new version of `@bpmn-io/properties-panel` to fix this.
> 
> #lastRelease

Unfortunately, it's not implemented in 0.7.0 :<
